### ISSUE TITLE
Trailing comma rule from prettier is not working

### ISSUE THREAD

**Issue by TotomInc** [prettier/eslint-plugin-prettier#206]
**What version of `eslint` are you using?**
5.16.0

**What version of `prettier` are you using?**
1.18.2

**What version of `eslint-plugin-prettier` are you using?**
3.1.0

**Please paste any applicable config files that you're using (e.g. `.prettierrc` or `.eslintrc` files)**
```javascript
// .eslintrc.js
module.exports = {
  root: true,
  env: {
    node: true
  },
  extends: ['plugin:vue/recommended', '@vue/airbnb', '@vue/typescript', 'plugin:prettier/recommended'],
  parserOptions: {
    parser: '@typescript-eslint/parser',
    sourceType: 'module',
    ecmaVersion: 2018
  },
  rules: {
    'import/prefer-default-export': 'off',
    'no-console': process.env.NODE_ENV === 'production' ? 'error' : 'off',
    'no-debugger': process.env.NODE_ENV === 'production' ? 'error' : 'off'
  }
};
```

```javascript
// .prettierrc.js
module.exports = {
  printWidth: 120,
  tabWidth: 2,
  tabs: false,
  semi: true,
  singleQuote: true,
  quoteProps: 'as-needed',
  tailingComma: 'all',
  bracketSpacing: true,
  jsxBracketSameLine: false,
  arrowParens: 'always'
};
```

**What source code are you linting?**
```typescript
// src/trailing-comma.ts
const obj = {
  test: 'hello',
};
```

**What did you expect to happen?**
ESLint should not show an error: `Delete "," eslint(prettier/prettier)` and keep the trailing comma according to my `.prettierrc.js` config.

**What actually happened?**
ESLint show me an error and always auto-fix it (in VS Code) when saving, but I want to keep all trailing comma as configured in the `.prettierrc.js` config.

**Comment by BPScott** [comment]
Your prettierrc says "tailingComma", it should be "trailingComma". Does that typo exist in your source code too?

**Comment by TotomInc** [comment]
Yes the typo exists in my source-code too. Sorry for creating a useless issue! 😄

**Comment by NgallaJuke** [comment]
This was not useless ! I used your .prettierrc.js file to solve my issue! When I saved a file, a comma is automaticly added to the of objects. So i got the warning everytime 
_**warning  Delete `,`  prettier/prettier**_ .

**Comment by Shadowmad** [comment]
Helped me as well. Thank you for creating this issue :)
